### ISSUE TITLE
Questions about functionality

### ISSUE THREAD

**Issue by KavanBahrami** [oculus-samples/Unreal-LayerSample#2]
README: "Actor_Blueprint illustrates rendering a widget into a stereo layer. The widget is first rendered into a material, then the SlateUI texture is pulled from the material into the stereo layer."

I don't see how this is happening. Can someone please clarify.

When the zip loads for me, the Actor_Blueprint in level has a texture assigned to the StereoLayer which is being displayed on PIE. The widget doesn't render to a material and the material is not being used in the StereoLayer.

Can someone on the team who built this please clarify desired intention and whether what I'm seeing is expected state of the project?

There seems to be some BP code missing.

**Comment by vokama** [comment]
Yes, this seems a bit misleading, thankfully the [Unreal Scene](https://github.com/oculus-samples/Unreal-Scene/) sample actually contains an example of rendering a UMG Widget into the Stereo Layer
